feature-utils/poly-look/src/react-components/filterChips.jsx

```diff
@@ -61,13 +61,26 @@ const FilterChips = ({
         )
   );
 
-  const [initialDefaultChips, setInitialDefaultChips] =
-    useState(defaultActiveChips);
+  const isFirstRender = useRef(true);
+  const initialActiveChips = useRef(defaultActiveChips);
 
-  if (defaultActiveChips !== initialDefaultChips) {
-    setActiveChips(defaultActiveChips);
-    setInitialDefaultChips(defaultActiveChips);
-  }
+  useEffect(() => {
```

Comment:
Are you sure this has to be in a `useEffect`? This code is executed after the initial rendering, but then you're causing a rerender anyway if you call `setActiveChips`.

Reply:
I see this in the documentation:

`Mutations, subscriptions, timers, logging, and other side effects are not allowed inside the main body of a function component (referred to as React’s render phase). Doing so will lead to confusing bugs and inconsistencies in the UI.` 

As the reason why one should use useEffect. And this code does mutate `initialActiveChips`.